src/libsync/propagatedownload.cpp

```diff
@@ -626,13 +623,13 @@ static QString makeRecallFileName(const QString &fn)
 
 void handleRecallFile(const QString& filePath, const QString& folderPath, SyncJournalDb& journal)
 {
-    qDebug() << "handleRecallFile: " << filePath;
+    qCDebug(lcPropagateDownload) << "handleRecallFile: " << filePath;
```

Comment:
The recall file is an important expert feature, i think the log should definitely show by default..